Gemfile

```diff
@@ -6,6 +6,8 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
+# fix activesupport to 7.0.8 due to a bug with >=7.1.0 in cocoapods (https://github.com/CocoaPods/CocoaPods/issues/12081)
+gem 'activesupport', '7.0.8'
```

Comment:
I run into this on a personal project and it didn't occur to me to mention it internally. I hope this didn't take too long to figure out.

Reply:
Same. Wrote a delayed post here: paNNhX-JX-p2